smclient/smclient.go

```diff
@@ -295,6 +296,12 @@ func (client *SMClient) register(config *config.Config) (err error) {
 		return aoserrors.Wrap(err)
 	}
 
+	if err := client.sendRuntimeInstanceNotifications(launcher.RuntimeStatus{
```

Comment:
We should send it in any case? Even if it is nil?

Reply:
no. Fixed